### PR TITLE
VR: fix wrong check when compare two configuration files

### PR DESCRIPTION
**Pull request by weizhouapache**
### Description

in some configuration files, the value can be set in multiple times and order matters. 
For example there are two haproxy configurations

```
listen 10_0_xx_yy-22
	bind 10.0.xx.yy:22
	balance roundrobin

listen 10_0_xx_yy-23
	bind 10.0.xx.yy:23
	balance source
```

```
listen 10_0_xx_yy-22
	bind 10.0.xx.yy:22
	balance source

listen 10_0_xx_yy-23
	bind 10.0.xx.yy:23
	balance roundrobin
```

without this change, they are considered as same configuration, which is obviously wrong.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI
- [ ] test (unit or integration test code)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [x] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by weizhouapache** [comment]
@blueorangutan package

**Review by sureshanaparti — APPROVED** [review]

**Comment by weizhouapache** [comment]
@blueorangutan test

**Review by rohityadavcloud — APPROVED** [review]

File: systemvm/debian/opt/cloud/bin/cs/CsFile.py [hunk 1]
```diff
@@ -175,6 +175,6 @@ def deleteLine(self, search):
         self.new_config = list(temp_config)
 
     def compare(self, o):
-        result = (isinstance(o, self.__class__) and set(self.config) == set(o.config))
+        result = (isinstance(o, self.__class__) and self.config == o.config)
```
Comment (DaanHoogland) [inline systemvm/debian/opt/cloud/bin/cs/CsFile.py:178]: So this means that we no longer compare the set to be equal but the line by line items (in order). Is that really what we want @weizhouapache ?

Comment (weizhouapache) [reply systemvm/debian/opt/cloud/bin/cs/CsFile.py:178]: yes, we should compare lines by lines.

the method is being used to compare the configurations for
- haproxy
- conntrackd
- radvd

for all 3, I think the order of configurations in the files matters.

**Review by DaanHoogland — APPROVED** [review]

**Pull request merged**